RELEASE_NOTES.md

```diff
@@ -6,64 +6,31 @@
+* See the [Historacle Design Doc](/docs/design_docs/011-historacle-pricing.md) for a description of how these prices are calculated.
+* See the [Leverage Module Spec](/x/leverage/README.md#historic-borrow-limit-value) for a description of how these prices are treated by the leverage protocol.
 
-Additional highlights:
-
-- Added `QueryMaxWithdraw` and `MsgMaxWithdraw` to allow user easily withdraw previously supplied tokens from the module back to the user balance.
-- Updated Cosmos SDK to v0.46.7
-
-Please see the [CHANGELOG](https://github.com/umee-network/umee/blob/v3.3.0/CHANGELOG.md) for an exhaustive list of changes.
-
-### Github Release
-
-Sinice `v3.2.0` new experimental features (disabled by default) are part of the linked binary. That changed the build process. Umeed officially doesn't support static CGO build (with `CGO_ENABLED=1`) any more. Github Actions only support build using Linux on amd64 -- we can not make a cross platform build using Github Actions (possible solution is to do it through Qemu emulator). So our Github release only contains source code archive and amd64 Linux binary.
-
-To run the provided binary, you **have to have `libwasmvm.x86_64.so v1.1.1`** in your system lib directory.
-
-Building from source will automatically link the `libwasmvm.x86_64.so` created as a part of the build process (you must build on the same host as you run the binary, or copy the `libwasmvm.x86_64.so` your lib directory).
+**Please Note** This upgrade requires the use of [Price Feeder V2.0.3](https://github.com/umee-network/umee/releases/tag/price-feeder%2Fv2.0.3) **AFTER** the Umee v4.0 Upgrade. Prior to this upgrade, you should stay on [Price Feeder V2.0.2](https://github.com/umee-network/umee/releases/tag/price-feeder%2Fv2.0.2).
 
-If you build on system different than Linux amd64, then you need to download appropriate version of libwasmvm (eg from [CosmWasm/wasmvm Relases](https://github.com/CosmWasm/wasmvm/releases)) or build it from source (you will need Rust toolchain).
+Additional highlights:
 
-Otherwise you have to download `libwasmvm`. Please check [Supported Platforms](https://github.com/CosmWasm/wasmvm/tree/main/#supported-platforms). Example:
+- [1694](https://github.com/umee-network/umee/pull/1694) `MsgMaxWithdraw`, `MsgMaxBorrow` and `MsgRepay` won't return errors if there is nothing to withdraw, borrow or repay respectively. Leverage `ErrMaxWithdrawZero` and `ErrMaxBorrowZero` has been removed.
 
-```bash
-wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/libwasmvm.$(uname -m).so -P /lib/
-```
+Please see the [CHANGELOG](/CHANGELOG.md#v4.0.0) for an exhaustive list of changes.
 
 ### Update instructions
 
-- Note: Skip this step if you build binary from source and are able to properly link libwasmvm.
-  - Download `libwasmvm`:
-
-```bash
-$ wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/libwasmvm.$(uname -m).so -O /lib/libwasmvm.$(uname -m).so
-```
```

Comment:
Ditto, let's keep the libwasmvm info.

Reply:
f6cfe968cebd905d5ca1661832ea182d9d9bb05d